### PR TITLE
install package manifest

### PR DESCRIPTION
**Pull request by ruffsl**
For osrf/docker_images#317.

### CODE REVIEW

File: sros2/setup.py [hunk 1]
```diff
@@ -11,17 +11,21 @@ def package_files(directory):
             paths.append(os.path.join('..', path, filename))
     return paths
 
-
```
Comment (jacobperron) [inline sros2/setup.py]: The linter is complaining about this change.

Comment (ruffsl) [reply sros2/setup.py]: @jacobperron , I've reverted this line.

**Comment by jacobperron** [comment]
Sorry for the delay.

@ruffsl Please do a squash-merge or rebase-merge next time to avoid an extra merge commit.

**Comment by ruffsl** [comment]
> Please do a squash-merge or rebase-merge next time to avoid an extra merge commit.

Can you set that as the default option for the repo in the github repo settings?
It's not apparent which megre style is prefered for every github repo otherwise.

**Comment by jacobperron** [comment]
I've updated the settings to disallow merge commits. I'm not sure if there's a reason we don't do this across the org, so I'll only touch the settings for this repo for now.

**Comment by mjcarroll** [comment]
I know that in some cases we don't do squash-merge to preserve the history.  The case that comes to mind is if there is a commit that represents a copy from somewhere else, and then additional commits for modifications to that copy.

It should also default to the last one that you used on that repository.

**Pull request merged**